src/terms/anyonic.jl

```diff
@@ -0,0 +1,91 @@
+# Ref https://arxiv.org/pdf/1901.10739.pdf
+# This term does not contain the kinetic energy, which must be added separately
+# /!\ They have no 1/2 factor in front of the kinetic energy,
+#     so for consistency the added kinetic energy must have a scaling of 2
+# Energy = <u, ((-i∇+βA)^2 + V) u>
+# where ∇∧A = 2π ρ, ∇⋅A = 0 => A = x⟂/|x|² ∗ ρ
```

Comment:
Can you write `x^⟂` just to make clear the `⟂` is not an operator, but the orthogonal component.

Reply:
Well I kind of meant at each occurrence of `x⟂` ...

Reply:
Again, it's an implementation comment of a pretty obscure feature... but fine. It doesn't drive you mad that I have sometiems ^2 and sometimes ²? :-p (I tried and figured out it was too annoying to type and didn't work in julia so gave up)